src/components/Button/Button.js

```diff
@@ -176,8 +176,8 @@ const getLabelTopMargin = (props) => {
   return getButtonFontSize(props) * 0.18;
 };
 
-const getButtonOpacity = ({ disabled, isLoading, theme }) => {
-  if (disabled && !isLoading) {
+const getButtonOpacity = ({ disabled, theme }) => {
+  if (disabled) {
```

Comment:
```suggestion
  if (disabled || isLoading) {
```

Reply:
Thought about this, but it's not the same as then it doesn't give control over button to show it as enabled while loadin. Hope it makes sense to leave it under control of `disabled` and `isLoading` separately which means `disabled` brings opacity down as it should and `isLoading` has it's purpose to show loading but doesn't have any effect on opacity.

Reply:
makes sense, you can keep the conditions as you changed it